src/main/java/pages/akaunting/AddNewCustomerPage.java

```diff
@@ -21,18 +24,16 @@ public class AddNewCustomerPage extends BasePageInteractions {
     private By zipCodeField = By.id("zip_code");
     private By stateField = By.id("state");
     private By countryField = By.cssSelector("input[placeholder='- Select Country -']");
-
+    private By canLoginChk = By.cssSelector("label[for='create_user']");
+    //WebElement canLoginChk = eventDriver.findElement(By.id("create_user"));
```

Comment:
Please remove commented lines like this.

Reply:
Done!